Prog/building permits_high school summary.sas

```diff
@@ -0,0 +1,59 @@
+/**************************************************************************
+ Program:  building permits_high school summary
+ Library:  Schools
+ Project:  NeighborhoodInfo DC
+ Author:   Yipeng Su
+ Created:  5/9/2018
+ Version:  SAS 9.4
+ Environment:  Windows
+ Modifications: 
+
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas"; 
+
+** Define libraries **;
+%DCData_lib( Schools )
+%DCData_lib( Mar)
+%DCData_lib (Realprop)
+%DCData_lib (DCRA)
+
+proc mapimport out=seniorhigh_map
+  datafile="L:\Libraries\Schools\Maps\CURRENT\School_Attendance_Zones_Senior_High.shp";  
+run;
+
+proc sort data=seniorhigh_map; by OBJECTID;
+run;
+
+goptions reset=global border;
+
+/*not working: says missing lat and lon variable
+proc gproject 
+ data = Realprop.Sales_res_clean
+ LATLON
+ out= work.Sales_res_clean_xy; 
+  id ssl;
+run;
+*/
+data newconstruction;
+set DCRA.Building_Permits_in_2017 (drop=objectid);
```

Comment:
Dropped _objectid_ from building permits since it is also ID var for map data set and having it both data sets may cause problems.